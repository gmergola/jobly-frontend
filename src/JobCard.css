.JobCard-card {
  padding: 10px;
}

.JobCard-title {
  color: rgb(75, 86, 139);
}

.JobCard-apply-btn {
  width: 100px;
  margin: 0 auto;
}