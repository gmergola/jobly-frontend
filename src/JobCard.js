import React from 'react';
import './JobCard.css';

/**JobCard: Child component to JobList
 * renders job information
 */
function JobCard({ title, salary, equity }) {
  return (
    <div className="JobCard-card card border-warning mb-3" >
      <div>
        <br />
        <h2 className="card-header"><b>Title</b>: {title}</h2>
        <div className="card-body text-warning">
          <h4 className="card-text"><b>Salary</b>: {salary}</h4>
          <h4 className="card-text"><b>Equity</b>: {equity}</h4>
        </div>
      </div>
    </div>
  );

}

export default JobCard