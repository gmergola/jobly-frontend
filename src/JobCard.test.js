import React from "react";
import { render } from "@testing-library/react";
import JobCard from "./JobCard";
import { MemoryRouter } from "react-router";


it("renders without crashing", function() {
  render(
    <MemoryRouter>
        <JobCard />
    </MemoryRouter>
  );
});

it("matches snapshot", function() {
  const { asFragment } = render(
    <MemoryRouter>
      <JobCard />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});