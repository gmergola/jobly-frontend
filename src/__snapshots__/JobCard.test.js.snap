// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="JobCard-card card border-info mb-3"
  >
    <div>
      <br />
      <h2
        class="card-header"
      >
        <b>
          Title
        </b>
        : 
      </h2>
      <div
        class="card-body text-info"
      >
        <h4
          class="card-text"
        >
          <b>
            Salary
          </b>
          : 
        </h4>
        <h4
          class="card-text"
        >
          <b>
            Equity
          </b>
          : 
        </h4>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
